<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceDlg</class>
 <widget class="QDialog" name="DeviceDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Device Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="deviceNameLabel">
         <property name="text">
          <string>Device Name:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="deviceNameLineEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="baudRateLabel">
         <property name="text">
          <string>Baud Rate:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="baudRateComboBox"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="dataBitsLabel">
         <property name="text">
          <string>Data Bits:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="dataBitsComboBox"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="parityLabel">
         <property name="text">
          <string>Parity:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="parityComboBox"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="stopBitsLabel">
         <property name="text">
          <string>Stop Bits:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="stopBitsComboBox"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="lineEndLabel">
         <property name="text">
          <string>Line End:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="lineEndComboBox"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="echoLabel">
         <property name="text">
          <string>Echo:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QCheckBox" name="echoCheckBox"/>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="postponeWritingLabel">
         <property name="text">
          <string>Postponed Writing:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QCheckBox" name="postponedWritingCheckBox"/>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="cmdOK">
         <property name="text">
          <string>OK</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cmdCancel">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
